### ISSUE TITLE
If an image fails to load (e.g. because of a broken link) then the loadedClass is never applied

### ISSUE THREAD

**Issue by fablrdigital** [fristys/masonry#13]
**Describe the bug**
The title says it all. I have a list of images being displayed in a masonry grid and one of them is broken (the image no longer exists). The entire grid is hidden until the grid is loaded and `initOnImageLoad` is set to true. The `loadedClass` is not applied to the grid since the images didn't successfully load.

**To Reproduce**
The following codes is used to initialise the list.

```
      const options = {
            gutter: 14,
            gutterUnit: 'px',
            loadedClass: 'c-grid--loaded',
            initOnImageLoad: true,
            columnBreakpoints: { 1150: 3, 980: 2, 450: 1 } };
        const masonry = new Masonry(this.element, options);
```

**Expected behavior**
When an image fails to load it should still be marked as completed and the `loadedClass` class should be applied.

**Desktop (please complete the following information):**
 - iOS
 - Chrome
 - Version 90.0.4430.212 (Official Build) (x86_64)

**Additional context**
Great plugin, by the way. :)

**Comment by fablrdigital** [comment]
It looks like this is happening because the `src` attribute is empty for some reason (maybe a quirk of the CMS I'm using). I suggest incrementing loaded image variable if the `src` attribute is empty so that loaded callback still gets triggered.

**Comment by fristys** [comment]
Hey @fablrdigital I see [where](https://github.com/fristys/masonry/blob/96b916bbb7e8c4bd144789f105706bbae8137bb3/src/utils/images-loaded.ts#L44) the problem might occur. It should be a fairly simple fix, and I'll try to get to it as soon as I get some free time. Thanks for your report :)

**Comment by fristys** [comment]
@fablrdigital This is now fixed in version `1.1.5`
